gameoflife.py

```diff
@@ -19,51 +18,71 @@
                                                               
                                                               '''
 
-def init_board(file_path):
-    b = []
+def init_board(file_path : str) ->list:
+    '''
+    Function initializes a board sccording to the content in file.
+    Input: file path.
+    Output: initialized board.
+    '''
+    board = []
     if not os.path.isfile(file_path):
         raise Exception(f"File {file_path} could not be found.")
     with open(file_path, mode="r") as file:
         file_content = file.readlines()
```

Comment:
please extract the file handling logic to a separate module (for example `FileReader`) and use it here

Reply:
I would like to get feedback on FileReader.py, to be sure I understood correctly.